ftw/testbrowser/core.py

```diff
@@ -1018,6 +1018,31 @@ def _normalize_url(self, url_or_object, view=None):
 
         return url
 
+    @staticmethod
+    def _correct_webdav_xml(html):
+        if isinstance(html, (unicode, str)) and 'xmlns:d="DAV:"' in html:
```

Comment:
Please add a short docstring explaning that we are fixing invalid WebDAV documents from Plone here.